### PR TITLE
Add webhook guide

### PR DESCRIPTION
**Pull request by BenjaminVanRyseghem**
First PR, please be gentle 🌹 :smile:

Todo:

- [x] add real data to the payload example

### CODE REVIEW

**Comment by Azdaroth** [comment]
@BenjaminVanRyseghem Hope I was gentle enough :D

**Comment by Azdaroth** [comment]
Taking over.

File: content/guides/webhook-subscriptions.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+### Application setting page
+
+First, go to [partner login page](https://www.bookingsync.com/en/partners/login).
+
+Once you are logged in, locate the application where you want to subscribe to webhooks and click the "Manage" button.
+
+### The "Webhooks" tab
+
+You should see a "Webhooks" tab
```
Comment (kaizencodes) [inline content/guides/webhook-subscriptions.md]: @Azdaroth I don't see the "Webhooks" tab. Is it released? checked the branches on core none seem to be about webhooks.

Comment (Azdaroth) [reply content/guides/webhook-subscriptions.md]: @kaizencodes It's currently hidden behind `webhooks_enabled` flag, we need the docs to get rid of it :P

**Pull request merged**